README.md

```diff
@@ -75,52 +80,52 @@ Content-Type: application/json
-Voorbeeld payload:
+  https://github.com/topicusonderwijs/toegang.org.examples/tree/master/examples/nodejs.
+
+At least, make sure the signature is verified and the timestamp in the `exp` (expires) field is in the future.
+
+## Step 4. Process information from the JWS
+
+A verified JWS contains the fields described below.
+
+| Field   | Mandatory | Omschrijving                                                                                                                  |
+| ------- | :-------: | ----------------------------------------------------------------------------------------------------------------------------- |
+| aud     |    Yes    | Unique identifier of the publisher's organization (UUID)                                                                      |
+| ean     |    Yes    | EAN of the product                                                                                                            |
+| exp     |    Yes    | Timestamp when the token expires (**milliseconds** since Unix epoch)                                                          |
+| ref     |    Yes    | Support code for the service desk to be able to follow the user through the process (XXXXXXXX - XXXXXXXX)                     |
+| sub     |    Yes    | ID of the account that is unique to the publisher (32 or 64 (= old) hex characters)                                           |
+| tlink   |    Yes    | TLink code of the license (XXXXXXXX)                                                                                          |
+| email   |    No     | Verified email of the user                                                                                                    |
+| extids  |    No     | Unique identifier of a user at other publishers (only supplied when sublicenses are used) - [{"orgid": ..., "externId": ...}] |
+| fn      |    No     | First name of the user                                                                                                        |
+| lac     |    No     | Linked accounts; array with historical identifiers of this user due to a merger or account merge, for example                 |
+| org     |    No     | ESN of the user's school or organization that includes that school (UUID)                                                |
+| orgname |    No     | Name of the user's school or organization that includes that school                                                      |
+| rol     |    No     | Role of the logged in user (eduPersonAffiliation)                                                                             |
+| rnd     |    No     | Random UUID to detect replay attacks                                                                                          |
+| endDate |    No     | _(coming soon)_ End date of the license (2020-04-01)                                                                          |
+
+The publisher should be able to log in to the user based on this data.
```

Comment:
Beetje verwarrende zin.
Ik zou er "The publisher should be able to log the user in based on this data." oid van maken.

Reply:
'should be able to log in the user' is denk ik correct